applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -533,6 +555,28 @@ static void usb_wakeup(void)
 	}
 }
 
+static void verify_selected_report_bitmask(void)
+{
+	BUILD_ASSERT((!USB_HID_SELECTIVE_SUB) ||
+		     (ARRAY_SIZE(usb_hid_selected_report_bitmask) ==
+		      CONFIG_USB_HID_DEVICE_COUNT));
+
+	BUILD_ASSERT(sizeof(usb_hid_selected_report_bitmask[0]) ==
+		     sizeof(usb_hid_device[0].report_bitmask));
+
+	uint16_t common_bitmask = 0;
+
+	for (size_t i = 0; i < CONFIG_USB_HID_DEVICE_COUNT; i++) {
+		/* USB HID instance that does not subscribe for any reports
+		 * should be removed. Given HID report can be handled only
+		 * by one USB HID instance.
```

Comment:
this last sentence is not true for dongle

Reply:
That could be misleading, I will rephrase it